REFERENCE.md

```diff
@@ -886,25 +886,28 @@ Default value: `$redis::params::minimum_version`
 
 ### `redis::administration`
 
-Allows various adminstrative settings for Redis
+Allows various administrative settings for Redis
 As documented in the FAQ and https://redis.io/topics/admin
 
+For disabling Transparent Huge Pages (THP), please use forge module:
+https://github.com/alexharv074/puppet-disable_transparent_hugepage
```

Comment:
Maybe link to the forge module instead??
https://forge.puppet.com/modules/alexharvey/disable_transparent_hugepage

Reply:
I personally haven't used this module. Do want to endorse it in this way or tone it down to "use separate module such as ..."?

Reply:
Oh, I used both of your recommendations here.